.travis.yml

```diff
@@ -5,28 +5,26 @@ addons:
 language: python
 sudo: false
 python:
-  - 3.5
+- 3.5
+- 3.8
 env:
-  - TOX_ENV=csslint
-  - TOX_ENV=eslint
-  - TOX_ENV=pycodestyle
-  - TOX_ENV=pylint
-  - TOX_ENV=py35-django111
-  - TOX_ENV=py35-django20
-  - TOX_ENV=py35-django21
-  - TOX_ENV=py35-django22
+- TOX_ENV=django22
+- TOX_ENV=csslint
+- TOX_ENV=eslint
+- TOX_ENV=pycodestyle
+- TOX_ENV=pylint
 before_install:
-  - "pip install -U pip"
-  - export BOTO_CONFIG=/dev/null
+- pip install -U pip
```

Comment:
I think that we should use the 20.0.2 version that we use in the other PRs

Reply:
done